src/components/mint/booking.cairo

```diff
@@ -0,0 +1,74 @@
+// Core imports
+
+use traits::{Into, TryInto, PartialEq};
+use option::OptionTrait;
+use debug::PrintTrait;
+
+#[derive(Drop, Copy)]
+enum BookingStatus {
+    Unknown: (),
+    Booked: (),
+    Failed: (),
+    Minted: (),
+    Refunded: (),
+}
+
+#[derive(Drop, Copy, Serde, starknet::Store)]
+struct Booking {
+    value: u256,
+    amount: u256,
+    status: u8,
```

Comment:
`Trait has no implementation in context: core::starknet::storage_access::Store::<carbon::components::mint::booking::BookingStatus>` even while derive Store.

I guess we could try by implementing manually Store, but it would just be more code to maintain and less safe imo.